python/ray/serve/_private/proxy.py

```diff
@@ -772,16 +779,6 @@ async def not_found_response(
 
         yield ResponseStatus(code=status_code, is_error=True)
 
-    async def timeout_response(self, request_id: str) -> ResponseGenerator:
-        status_code = 408
-        for message in convert_object_to_asgi_messages(
-            f"Request {request_id} timed out after {self.request_timeout_s}s.",
-            status_code=status_code,
-        ):
-            yield message
-
-        yield ResponseStatus(code=status_code, is_error=True)
```

Comment:
This was quite awkward because the `ResponseStatus` was being duplicated (and double yielded)